core/domain/classifier_domain.py

```diff
@@ -291,6 +291,23 @@ def to_dict(self):
             'data_schema_version': self._data_schema_version
         }
 
+    def to_player_dict(self):
+        """Constructs a dict containing a training job domain object's algorithm_id,
+        classifier_data and data_schema_version.
+
+
+        Returns:
+             A dict containing training job domain object's algorithm_id,
+             classifier_data and data_schema_version.
+        """
+
+        return {
+            'algorithm_id': self._algorithm_id,
+            'classifier_data': self._classifier_data,
+            'data_schema_version': self._data_schema_version
+        }
+
```

Comment:
Drop extra newline (we have 2 lines separating top-level classes/functions, 1 line separating methods within a class).

Reply:
I don't think this is done, I still see 2 blank lines?